.github/workflows/tests-pydev.yml

```diff
@@ -4,11 +4,11 @@ on:
   push:
     branches:
       - master
-      - pydev_9_3_1
+      - pydev_9_3
```

Comment:
Agreed that this should really not be part of this PR. I'll change it separately.

Reply:
Okay, I will revert that commit.